packages/grid/_modules_/grid/hooks/core/preProcessing/gridPreProcessingApi.ts

```diff
@@ -5,6 +5,8 @@ export enum GridPreProcessingGroup {
   scrollToIndexes = 'scrollToIndexes',
   columnMenu = 'columnMenu',
   canBeReordered = 'canBeReordered',
+  registerFiltering = 'registerFiltering',
```

Comment:
Why not? "register" confuses with the name of hook `useGridRegisterPreProcessor`.

```suggestion
  filterMethod = 'filterMethod',
```

similar for the sorting.